src/Native/Runtime/arm64/PInvoke.S

```diff
@@ -1,4 +1,355 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-// TODO: Implement
+
+#include <unixasmmacros.inc>
+#include "AsmOffsets.inc"
+
+.global RhpTrapThreads
+
+// Note: these must match the defs in PInvokeTransitionFrameFlags defined in rhbinder.h
+PTFF_SAVE_X19           = 0x00000001
```

Comment:
Only two of the PTFF constants are used here. Include just the two that are actually used here?